dbt/adapters/athena/lakeformation.py

```diff
@@ -33,6 +36,7 @@ def __init__(self, lf_client: LakeFormationClient, relation: AthenaRelation, lf_
         self.table = relation.identifier
         self.lf_tags = lf_tags_config.tags
         self.lf_tags_columns = lf_tags_config.tags_columns
+        self.lf_inherited_tags = set(lf_tags_config.inherited_tags if lf_tags_config.inherited_tags else [])
```

Comment:
```suggestion
        self.lf_inherited_tags = set(lf_tags_config.inherited_tags)
```
it already has default value, so no need for some extra check I guess

Reply:
I agree, there is a default value already, and also, if we follow this: https://github.com/dbt-athena/dbt-athena/pull/453/files#r1358873411 we don't need extra convertion from list to set.

Reply:
Good point!